components/org.wso2.identity.webhook.common.event.handler/src/main/java/org/wso2/identity/webhook/common/event/handler/EventConfigManager.java

```diff
@@ -46,7 +46,7 @@ public class EventConfigManager {
     private final ResourceConfig eventSchema;
     private static volatile EventConfigManager instance;
 
-    private EventConfigManager() throws IdentityEventServerException {
+    EventConfigManager() throws IdentityEventServerException {
```

Comment:
We need to check why this default constructor is needed. There are better ways to handle loading of these files. Approving for now